src/index.ts

```diff
@@ -0,0 +1,29 @@
+import './main.scss';
+import { sendRequest } from './modules/postToDo';
+import { createLi } from './modules/createToDo';
+import { allDelete } from './modules/allDelete';
+import { sendToDo } from './modules/postToDo';
+
+export type Item = {
+    completed: boolean,
+    id: number,
+    title: string,
+    userId: number,
+}
+
+sendRequest<Item[], void>('GET', "todos")
```

Comment:
Второй дженерик необязательно указывать, если ты его не используешь 

```suggestion
sendRequest<Item[]>('GET', "todos")
```

Reply:
аналогично везде